paimon-web-ui/src/views/cdc/components/list/use-table.ts

```diff
@@ -15,106 +15,113 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License. */
 
-import { useCDCStore } from '@/store/cdc';
-import type { Router } from 'vue-router';
-import TableAction from '@/components/table-action';
-import { deleteCdcJobDefinition, getCdcJobDefinition, listAllCdcJob } from '@/api/models/cdc';
-import { get } from 'lodash';
-export const useTable = () => {
-    const router: Router = useRouter()
-    const { t } = useLocaleHooks()
-    const tableVariables = reactive({
-        columns: [
-            {
-                title: t('cdc.job_name'),
-                key: 'name',
-                resizable: true
-            },
-            {
-                title: t('cdc.synchronization_type'),
-                key: 'cdcType',
-                resizable: true,
-                render: (row:any)=>{
-                    if(row.cdcType == 0){
-                        return t('cdc.single_table_synchronization')
-                    }else if(row.cdcType ==1){
-                        return  t('cdc.whole_database_synchronization')
-                    }
-                }
-            },
-            {
-                title: t('cdc.job_description'),
-                key: 'description',
-                resizable: true
-            },
-            {
-                title: t('cdc.create_user'),
-                key: 'createUser',
-                resizable: true
-            },
-            {
-                title: t('cdc.create_time'),
-                key: 'createTime',
-                resizable: true
+import { useCDCStore } from '@/store/cdc'
+import type { Router } from 'vue-router'
```

Comment:
sort the import. Of course, we will fix it later in the process of eslint improvement. This is just a reminder.